src/Symfony/Component/Serializer/Normalizer/GetSetMethodNormalizer.php

```diff
@@ -36,6 +41,25 @@ class GetSetMethodNormalizer extends AbstractObjectNormalizer
 {
     private static $setterAccessibleCache = [];
 
+    private $allowNormalizationOfObjectsWithoutAnyGetters;
+
+    public function __construct(
+        ClassMetadataFactoryInterface $classMetadataFactory = null,
+        NameConverterInterface $nameConverter = null,
+        PropertyTypeExtractorInterface $propertyTypeExtractor = null,
+        ClassDiscriminatorResolverInterface $classDiscriminatorResolver = null,
+        callable $objectClassResolver = null,
+        array $defaultContext = [],
+        bool $allowNormalizationOfObjectsWithoutAnyGetters = false,
+    ) {
+        parent::__construct($classMetadataFactory, $nameConverter, $propertyTypeExtractor, $classDiscriminatorResolver, $objectClassResolver, $defaultContext);
+        $this->allowNormalizationOfObjectsWithoutAnyGetters = $allowNormalizationOfObjectsWithoutAnyGetters;
+
+        if (\func_num_args() < 7) {
+            trigger_deprecation('symfony/serializer', '6.1', '$allowNormalizationOfObjectsWithoutAnyGetters parameter of %s::__construct() should be explicitly provided since the default value will change to `true` in symfony/serializer >=7.0', __CLASS__);
```

Comment:
```suggestion
            trigger_deprecation('symfony/serializer', '6.2', '$allowNormalizationOfObjectsWithoutAnyGetters parameter of %s() should be explicitly provided since the default value will change to `true` in symfony/serializer >=7.0', __METHOD__);
```